app/views/document_topics/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :back_link, document_path(@document) %>
+<% content_for :title, t("document_topics.edit.title", title: @document.title_or_fallback) %>
+
+<% def unroll(topic) %>
+  <% capture do %>
+    <li>
+      <%= topic[:title] %>
+
+      <ul>
+        <% topic[:children].each do |child_topic| %>
```

Comment:
Can this sometimes create an empty list on the last item when `topic[:children]` is empty?

Reply:
It seems like the 'child_taxons' link is either present or not, so this shouldn't happen. But I agree it's logical to see the code there anyway ;-).